app/controllers/items_controller.rb

```diff
@@ -98,14 +95,15 @@ def create
   end
   
   def search
-    if !params[:search].empty?
+    if params[:search] == nil
+      q = params[:q].downcase
+      @item = Item.search(q)
+      render json: @item.select(:name)
+    else
```

Comment:
Did u cover this in tests (if it broke anything), i see you changed one test line but dont see anything else

Reply:
Yep, I made sure all tests work